.example.env.development

```diff
@@ -2,6 +2,7 @@
 SOLR_PORT=8983
 SOLR_URL=http://127.0.0.1:8983/solr/blacklight-core
 SOLR_VERSION=9.1.1
+OGM_PATH=tmp/opengeometadata
```

Comment:
Adding this (and also to .example.env.production!) to remind us that the OGM_PATH is different in Development and Production.